dataprofiler/labelers/labeler_utils.py

```diff
@@ -79,7 +79,7 @@ def evaluate_accuracy(
     predicted_entities_in_index: List[List[int]],
     true_entities_in_index: List[List[int]],
     num_labels: int,
-    entity_rev_dict: Dict,
+    entity_rev_dict: Dict[int, Any],
```

Comment:
I believe the `Any` is required to be `str`

Reply:
Makes sense. Made the change.